cmd/simulator/load/tx_generator.go

```diff
@@ -8,15 +8,18 @@ import (
 	"crypto/ecdsa"
 	"fmt"
 
+	"github.com/ava-labs/subnet-evm/cmd/simulator/txs"
 	"github.com/ava-labs/subnet-evm/core/types"
 	"github.com/ava-labs/subnet-evm/ethclient"
 	ethcrypto "github.com/ethereum/go-ethereum/crypto"
 )
 
+var _ txs.TxSequence[*types.Transaction] = (*txSequence)(nil)
+
 type CreateTx func(key *ecdsa.PrivateKey, nonce uint64) (*types.Transaction, error)
 
 // GenerateTxSequence fetches the current nonce of key and calls [generator] [numTxs] times sequentially to generate a sequence of transactions.
-func GenerateTxSequence(ctx context.Context, generator CreateTx, client ethclient.Client, key *ecdsa.PrivateKey, numTxs uint64) ([]*types.Transaction, error) {
+func GenerateTxSequence(ctx context.Context, generator CreateTx, client ethclient.Client, key *ecdsa.PrivateKey, numTxs uint64) (txs.TxSequence[*types.Transaction], error) {
```

Comment:
Do we need this to be exist/exported? I think `GenerateTxSequences` is just a generalized version of this.

Reply:
i can unexport this

Reply:
never mind it is used in funder